### PR TITLE
chore: Prepare 1.230.0 release

### PR DESCRIPTION
**Pull request by patrickhousley**
### Overview

Updating docs and version for 1.230.0 release.

### Related Issue(s)

N/A

### Testing

N/A

### CODE REVIEW

**Review by metal-messiah — APPROVED** [review]

File: CHANGELOG.md
Comment (cwli24) [inline CHANGELOG.md:1]: `setCustomAttribute` update memo missing in 1.230

File: package.json [hunk 1]
```diff
@@ -134,6 +134,7 @@
     "url": "https://github.com/newrelic/newrelic-browser-agent.git"
   },
   "dependencies": {
+    "caniuse-lite": "^1.0.30001477",
```
Comment (metal-messiah) [inline package.json:137]: Should this be a dev-dependency?

Comment (patrickhousley) [reply package.json:137]: yes

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,18 +5,27 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
...
 
 ### Fix npm package exports
-Fixed some bad exports in the npm package.
+Removed some dangling, unused, and non-existent imports and exports from the source code that was causing issues with consuming the NPM package in some projects.
 
 ### Fix npm package import for typescript projects
 Fixed an issue where typescript projects attempting to import the browser agent library would not get proper auto-complete in the import statement and would receive module declaration errors. The browser agent will not use tsc to generate type definitions from the JavaScript code at build time. The package.json has been updated with appropriate type export declarations.
 
+### Add error catch around sendBeacon
+Added error catch logic around the sendBeacon usage. In the event sendBeacon throws an error during the final harvest, the final harvest will attempt to fallback to image data submission method.
```
Comment (bjfield) [inline CHANGELOG.md:24]:
```suggestion
Added error catch logic around the sendBeacon usage. In the event sendBeacon throws an error during the final harvest, the final harvest will attempt to fall back to the image data submission method.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,18 +5,27 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
...
+
+### Removed array reduce polyfill
+Removed the array reduce polyfill in favor of native browser array reduce method.
```
Comment (bjfield) [inline CHANGELOG.md:27]:
```suggestion
Removed the array reduce polyfill in favor of the native browser array reduce method.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,18 +5,27 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ## v1.230.0
 
-### Add setUserId and enhance setErrorHandler APIs
-`setUserId` is an extension of setCustomAttribute that tags a user ID attribute to all payloads. `setErrorHandler` now allows the provided callback to fingerprint and group errors by returning an object with a `group` property.
+### Add persistence for custom attributes
+Added the ability for custom attributes to be persisted between page loads via session storage. The API `setCustomAttribute` takes a new, third parameter. The new parameter defaults to `false`. When passed a `true` boolean, the provided attribute will be persisted in session storage and restored on subsequent page loads. Session storage is subject to the [same-origin policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) of the browser.
+
+### Add setUserId API
+Added a new API method `setUserId` that defines a custom attribute containing the provided user ID. The user ID will be included in all payloads. The `setUserId` API is an extension of the `setCustomAttribute` API and automatically persists between page loads.
```
Comment (bjfield) [inline CHANGELOG.md:12]:
```suggestion
A new `setUserId` API method is now available. It defines a custom attribute containing a specified user ID, to be included in all payloads. The `setUserId` API is an extension of the `setCustomAttribute` API and automatically persists between page loads.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,18 +5,27 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ## v1.230.0
 
-### Add setUserId and enhance setErrorHandler APIs
-`setUserId` is an extension of setCustomAttribute that tags a user ID attribute to all payloads. `setErrorHandler` now allows the provided callback to fingerprint and group errors by returning an object with a `group` property.
+### Add persistence for custom attributes
+Added the ability for custom attributes to be persisted between page loads via session storage. The API `setCustomAttribute` takes a new, third parameter. The new parameter defaults to `false`. When passed a `true` boolean, the provided attribute will be persisted in session storage and restored on subsequent page loads. Session storage is subject to the [same-origin policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) of the browser.
```
Comment (bjfield) [inline CHANGELOG.md:9]:
```suggestion
Custom attributes may now optionally be persisted between page loads via session storage. The API method `setCustomAttribute` now takes a third parameter, which defaults to `false`. When `true`, this boolean indicates that the provided custom attribute should be persisted in session storage and restored on subsequent page loads. Session storage is subject to the [same-origin policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) of the browser.
```

**Review by bjfield — APPROVED** [review]

**Pull request merged**